config/create_help.txt

```diff
@@ -118,15 +122,15 @@ Create Snapshot Schedule:
                                 2. To create snapshot on 5th, 15th and 25th of month, specify x as "0 * 5,15,25 * *"
                                 3. To create snapshot every quarter, specify x as "0 * * 3,6,9,12 *"
                                 4. To create snapshot on Monday, Wednesday and Friday, specify x as "0 * * * 1,3,5"
--o scheduleName=x               This option is mandatory. x is a string which indicates name for the schedule on 3PAR.
-                                Note: When this parameter is passed with string 'auto' , then the scheduleName is
-                                auto generated with a timestamp. This is to support kubernetes environment where 
+-o scheduleName=x               This option is mandatory. x is a string which indicates name for the schedule on 3PAR/PRIMERA.
+                                Note: When this parameter is passed with string 'auto’, then the scheduleName is
```

Comment:
@bhagyashree-sarawate Here, the quotes used for closing auto is not matching with opening quote. Please correct the typo.

Reply:
done